tools/pdf2txt.py

```diff
@@ -39,15 +44,8 @@ def extract_text(files=[], outfile='-',
     else:
         laparams = None
 
-    imagewriter = None
-    if output_dir:
-        imagewriter = ImageWriter(output_dir)
-
```

Comment:
Duplicate https://github.com/pdfminer/pdfminer.six/blob/develop/pdfminer/high_level.py#L65-L67

Reply:
It took me a while to understand that it is not only duplicate code (hence, a function could be extracted), but also that the `imagewriter` variable is unused.